structure/templates/ligand_complex_models.html

```diff
@@ -0,0 +1,569 @@
+
+
+
+                                        <!--               LIGAND            -->
+
+                    {% for ligand_object in model.prefetch_ligands %}
+                                        <!-- ligand name 7 -->
+                        <td> <a href="/ligand/{{ ligand_object.ligand.id }}/info">{{ ligand_object.ligand.name | upper }}</a>  </td>
+
+                                            <!-- ligand type 8 -->
+                        {% if ligand_object.ligand.ligand_type.slug == "small-molecule" %}
+                            <td>Small mol</td>
+                        {% else %}
+                            <td> {{ ligand_object.ligand.ligand_type.slug | title}} </td>
+                        {% endif %}
+                    {% endfor %}
+
+
+                                        <!--               RECEPTOR            -->
+
+                    
+                    <!-- Gene name 3 -->
+                    <td><a target="_blank" href="http://www.uniprot.org/uniprot/{{ model.protein_conformation.protein.accession }}">{{ model.gene_name }}</a></td>
+
+                    <!-- Protein name 4 -->
+                    <td> <a href=" /protein/{{model.protein_conformation.protein.entry_name}} ">{{ model.protein_conformation.protein.name|receptor_short|safe }}</a></td>
+
+                    <!-- Fam 5 -->
+                    <td>{{ model.protein_conformation.protein.family.parent.name }}</td>
+
+                    <!-- Class 6-->
+                    <td>{{ model.protein_conformation.protein.family.parent.parent.parent.name|cut_classname }}</td>
+
+                    <!-- HETEROTRIMMER -->
+                    <!-- {% if model.signprot_complex %}
+                        <td>Yes</td>
+                    {% else %}
+                        <td>No</td>
+                    {% endif %} -->
+
+
+
+                    <!--                         Scores                           -->
+
+                    {% if model.prefetch_af_scores %}
+                        <td>{{ model.prefetch_af_scores.0.pae_mean }} </td>
+
+                        <!-- <td>{{ model.pk }} </td> -->
+                    {% else %}
+                        <td> - </td>
+                    {% endif %}
+                    {% if model.prefetch_rfaa_scores %}
+                        <td>{{ model.prefetch_rfaa_scores.0.plddt_mean }} </td>
+
+                        <!-- <td>{{ model.pk }} </td> -->
+                    {% else %}
+                        <td> - </td>
+                    {% endif %}
+
+
+                    <!--                         DATE                           -->
+                     <td> {{ model.publication_date }} </td>
+
+                </tr>
+            {% endfor %}
+            </tbody>
+        </table>
+    </div>
+</div>
+<div id="superposition-modal" class="modal">
+  <!-- Modal content -->
```

Comment:
this we can keep here commented as I am pretty sure we will need to add this functionality in the near future